gapis/api/api.go

```diff
@@ -57,6 +58,9 @@ type API interface {
 	// The segments of memory that were used to create these commands are
 	// returned in the rangeList.
 	RebuildState(ctx context.Context, s *GlobalState) ([]Cmd, interval.U64RangeList)
+
+	// GetFramegraph returns the framegraph of the capture.
+	GetFramegraph(ctx context.Context, p *path.Capture) (*Framegraph, error)
```

Comment:
Since you are resolving the capture in order to get the API instance, passing in the capture path here is useless, please remove it.

Reply:
The path is needed in the implementation of `GetFramegraph` for two things:
- create the dependency graph
- sync.MutateWithSubcommands

Do you prefer to pass the capture and re-create its path inside `GetFramegraph`?